test/functional/apps/discover/group1/_filter_editor.ts

```diff
@@ -103,7 +103,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
             body: {
               version: '2.0.0',
             },
-            refresh: 'wait_for',
```

Comment:
This fixes a test, but I'm also curious: why, or ... why did it work before. given we tell elasticsearch not to refresh the index instantly?